detail.php

```diff
@@ -7,8 +7,8 @@
 use TheRealMVP\DisplayHelpers\DisplayData;
 
 require_once './vendor/autoload.php';
+require_once 'importer.php';
```

Comment:
Why do we need the importer in here? This will clear the DB and re-fetch the API every time you view a detail page

Reply:
Removed